src/js/components/MaskedInput/MaskedInput.js

```diff
@@ -124,6 +124,7 @@ const dropAlign = { top: 'bottom', left: 'left' };
 const MaskedInput = forwardRef(
   (
     {
+      a11yTitle,
```

Comment:
Needs to be added to MaskedInput documentation.

Reply:
yes, added